js/app.jsx

```diff
@@ -159,6 +187,106 @@ var SearchableRemoteListMixin = {
 
 }
 
+var ListToolbar = React.createClass({
```

Comment:
I would prefer different class, because the content is totally different (and will be much more different after we add more features).